ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/PreviewProjectTemplate.cshtml.cs

```diff
@@ -17,15 +17,12 @@ namespace ApplyToBecomeInternal.Pages.TaskList
 	public class PreviewHTBTemplateModel : BaseAcademyConversionProjectPageModel
 	{
 		private readonly KeyStagePerformanceService _keyStagePerformanceService;
-		private readonly ILegalRequirementsRepository _legalRequirementsRepository;
 		private readonly ErrorService _errorService;
 
 		public PreviewHTBTemplateModel(KeyStagePerformanceService keyStagePerformanceService, 
-			IAcademyConversionProjectRepository repository, ErrorService errorService,
-			ILegalRequirementsRepository legalRequirementsRepository) : base(repository)
+			IAcademyConversionProjectRepository repository, ErrorService errorService) : base(repository)
```

Comment:
Couple of warnings. 
![image](https://user-images.githubusercontent.com/109524605/192537786-f1e0f089-f3a3-474d-84d1-9593a87f0b61.png)